src/react/tests/useIsError.test.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+
+import { queryKey, renderWithClient } from './utils'
+import { useQuery, useIsError, QueryClient } from '../..'
+
+describe('useIsError', () => {
+  xit('should return number of errored queries', async () => {
+    const queryClient = new QueryClient()
+
+    const key1 = queryKey()
+
+    function Page() {
+      const { status } = useQuery(key1, async () => {
+        throw new Error()
```

Comment:
I would appreciate help in getting this to work.

It retries three times and then the test exits. Status remains 'loading'

Reply:
you probably need to turn off retries for the query.
---
but let's please continue [the discussion](https://github.com/tannerlinsley/react-query/discussions/2524) to see if there aren't better ways to handle your issue. I don't think that a hook that gives you a number of failed queries is particularly useful.